src/main/kotlin/eu/europa/ec/eudi/openid4vci/internal/http/AuthorizationEndpointClient.kt

```diff
@@ -151,6 +151,7 @@ internal class AuthorizationEndpointClient(
                 issuerState?.let { customParameter("issuer_state", issuerState) }
                 if (scopes.isNotEmpty()) {
                     scope(NimbusScope(*scopes.map { it.value }.toTypedArray()))
+                    resource(credentialIssuerId.value.value.toURI())
```

Comment:
Populate the `resource` parameter in case credential issuer is not authorization server.
To do so, please use

```kotlin
if (!isCredentialIssuerAuthorizationServer()){
   // add resource parameter
}
```

Check my last comment about the `isCredentialIssuerAuthorizationServer()`

Reply:
Done in 895197f4635224c857d46db5a10f74209307e8d3.